client/store/cart.js

```diff
@@ -57,8 +57,15 @@ export const getFromDB = () => async dispatch => {
 
 export const createOrder = () => async (dispatch, getState) => {
   const total = getState().total;
-  await axios.post('/api/orders', { amount: parseFloat(total) });
+  const cart = getState().cart;
+  const order = await axios.post('/api/orders', { amount: parseFloat(total) });
+  const orderId = order.data.id;
+  cart.forEach(function(item) {
+    item.orderId = orderId;
+  });
+  await axios.put('/api/carts', cart);
```

Comment:
when order is created, add an "orderId" key to cart items on state, then do a PUT request for those items, which will add an "orderId" column (or create an instance with an "orderId" column, if no instance already existed)